app/services/gpo_reminder_sender.rb

```diff
@@ -1,9 +1,13 @@
 class GpoReminderSender
   def send_emails(for_letters_sent_before)
+    letter_eligible_range =
+      IdentityConfig.store.usps_confirmation_max_days.days.ago..for_letters_sent_before
+
     profiles_due_for_reminder = Profile.joins(:gpo_confirmation_codes).
       where(
-        gpo_verification_pending_at: ..for_letters_sent_before,
+        gpo_verification_pending_at: letter_eligible_range,
         gpo_confirmation_codes: { reminder_sent_at: nil },
+        deactivation_reason: nil,
```

Comment:
I think this will skip users that are also going through in person verification. If we wait to merge until the transition to in_person_verification_pending_at is complete, this will work as intended. Otherwise we need to include `deactivation_reason: in_person_verification_pending`.

Reply:
Adding a test case to catch this and code to handle it.

Reply:
Added.

Reply:
Nevermind, I see now. The `with_pending_in_person_enrollment` on the user factory creates a profile with the deactivation reason.